test/http-mock-server.h

```diff
@@ -112,7 +115,9 @@ class HttpMockServer {
     EXPECT_FALSE(error) << error.message();
 
     acceptor_.bind(
-        asio::ip::tcp::endpoint(asio::ip::tcp::v4(), 0),
+        asio::ip::tcp::endpoint(
+            asio::ip::address_v4::from_string("127.0.0.1"),
+            0),
```

Comment:
I'd like to add a comment here explaining why we can't use the equivalent of  `INADDR_ANY` and must specify the loopback IP address on Windows.

Also, should we use `asio::ip::address_v1::loopback()`?

Reply:
Are you sure we need a comment on why we should use loopback and not open up our application for any incoming connection? :D

Reply:
I added a comment!